db/fixtures/miq_shortcuts.yml

```diff
@@ -217,7 +217,7 @@
 - :name: containers
   :description: Compute / Containers / Containers
   :url: /container/show_list
-  :rbac_feature_name: containers
+  :rbac_feature_name: container_show_list
   :startup: true
```

Comment:
I expected to see 'container' here. Why this change?

It breaks the UI. I missed that in the review :-(